app/admin/users.rb

```diff
@@ -4,13 +4,18 @@
   permit_params :email, :password, :admin
 
   before_action :remove_password_params_if_blank, only: [:update]
+  after_action :send_email, only: [:create]
```

Comment:
Why not using a `after_create` callback in the model? If we use it we do not need to do this here and in the controller.

Reply:
I need to send the email from the creator email, and using after_create I probably need to change the model to save that email to use it, right? 
Do you think it is important to do this now? Anyway, I understand your approach and think it is better